independent-projects/qute/debug/src/main/java/io/quarkus/qute/debug/agent/condition/ConditionalExpressionHelper.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.qute.debug.agent.condition;
+
+import io.quarkus.qute.Engine;
+import io.quarkus.qute.TemplateNode;
+
+public class ConditionalExpressionHelper {
+
+    private static final Engine conditionEngine;
+
+    static {
+        conditionEngine = Engine.builder().addDefaults().build();
+    }
+
+    public static TemplateNode parseCondition(String condition) {
```

Comment:
@mkouba I need to evaluate Qute condition expression:

 * to manage conditonal breakpoint
 * evaluate a conditional expression when user evaluate an expression with debugger

Qute doesn't provide this mechanism. My hack  is to create a template with `#if` and get this ifNode.